packages/@stylexjs/postcss-plugin/src/plugin.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+const postcss = require('postcss');
+const createBuilder = require('./builder');
+
+module.exports = function createPlugin() {
```

Comment:
`bundler` keeps track of files internally.

I moved the code from `src/index.js` to `src/plugin.js` and made it a factory so that each test rcreates the plugin so that no state leaks over to other tests. There are no other behavior changes in this file.